### DISCUSSION TITLE
Forbidden - you don't have permissions to access this resource - Apache/2.4.46 (Unix) Server at www.nobulrs.com Port 80

### DISCUSSION THREAD

**Discussion by mattnobul** [bullhorn/career-portal#414] [Q&A] [answered]
## Current behavior

- I have integrated the bullhorn plug-in onto our word press website but now I get "Apache/2.4.46 (Unix) Server at www.nobulrs.com Port 80" error message when I load up the page. I have checked previous comments about having an index page and checked on 123 reg and can confirm there is an index.php there. The page it should be loading on is - www.nobulrs.com/jobs

## Expected behavior

- The customer portal loads correctly.

## Screenshots

![image](https://user-images.githubusercontent.com/88542808/128510235-b1c5889c-04b3-473f-89c9-4e598f70185b.png)

**Comment by charlesabarnes** [comment, marked as answer]
index.html has to be added to the list of default index pages, not just index.php.  You can view that it works by navigating directly to the iframe source and adding index.html to it. https://www.nobulrs.com/site/wp-content/plugins/bullhorn-oscp/index.html

**Reply by mattnobul** [reply]
Hi Charles, I'm confused the link you posted say's it works - but when I go back to my jobs page it doesn't work?

**Reply by charlesabarnes** [reply]
Yes, you have to add index.html as a default index document in your web host

**Reply by mattnobul** [reply]
I have spoken to my web host and he said this "As far as I'm concerned, the 123 Reg hosting package already comes with an index.html file by default which should be present in your website’s root directory."

**Reply by charlesabarnes** [reply]
Depending on your host, this is what I am referring to 
https://hostadvice.com/how-to/how-to-change-default-index-page-in-htaccess-from-cpanel/

Yes, the page already exists, but when you navigate to a directory on a web host, your web host has to know what to do by default.  Some hosts just list all of the files, some automatically redirect you to a index.php file, but in this case in the bullhorn-oscp subdirectory, it needs to redirect to a index.html file and it is not doing that currently.   This is a server configuration issue.    as navigating to https://www.nobulrs.com/site/wp-content/plugins/bullhorn-oscp/ should work as it does on http://charlesabarnes.com/bullhornWordpress/wp-content/plugins/bullhorn-oscp/ for example

**Reply by mattnobul** [reply]
Hi Charles,

Our host are 123reg, unfortuantely I have no way of accessing the document route folder and they are refusing to help from their side as they said the index.html file is there. Have you got any suggestions?

**Reply by mattnobul** [reply]
Hi Charles, I have followed your instructions and I have no taken the whole website down...

http://www.nobulrs.com

**Reply by charlesabarnes** [reply]
I do suggest restoring your configuration as it was previously.   I also suggest ensuring the changes to your configuration that are made are made by a system administrator proficient with your website's setup.      There are too many nuances with web administration to provide a step-by-step guide with your specific web host on your specific platform.

**Reply by mattnobul** [reply]
Hi Charles,

I have taken it out and now my site is continuously trying to route to index.html and crashing

**Reply by charlesabarnes** [reply]
As I dont have the tools required to diagnose and help with the issue and as I don't know what steps were specifically performed, I need to advise that you work with your web host support team or site administrators.

**Reply by pbearne** [reply]
@mattnobul If can't sort this reach to us as we have WordPress base solution  https://matadorjobs.com/

**Comment by khaif-maker** [comment]
i am getting the same message "you dont have permission to access this resourse " what can i do can you give me some detailed steps
